icinga2/resource_icinga2_host.go

```diff
@@ -30,6 +30,14 @@ func resourceIcinga2Host() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"groups": &schema.Schema{
+				Type:     schema.TypeList,
+				Required: true,
```

Comment:
I would prefer that groups be optional, like templates. go-icinga2-api has been modified ( v0.2.x ) to support groups being optional.